index.bs

```diff
@@ -4741,9 +4740,15 @@ where [=map/keys=] are strings and [=map/values=] are of a particular type speci
     };
 </pre>
 
-Dictionaries are always passed by value.  In language bindings where a dictionary is represented by an object of some kind, passing a
-dictionary to a [=platform object=] will not result in a reference to the dictionary being kept by that object.
-Similarly, any dictionary returned from a platform object will be a copy and modifications made to it will not be visible to the platform object.
+Dictionaries are always passed by value: the dictionary does not retain a reference to its
+language-specific representation (e.g., the corresponding ECMAScript object). So for example,
+returning a dictionary from an [=operation=] will result in a new ECMAScript object being created
```

Comment:
I don't think the returning part is what should be clarified, as returning dictionaries is rare... it's more the passing in a dictionary in from JS to a method that is "deeply copied" (passed by value) that should be clarified... most JS programmers won't know what "passed by value" means, but it's second nature to all c++ and Rust programmers.

Reply:
Did you see the next sentence?

(BTW it's shallowly copied, not deeply.)

Reply:
yep, next sentence is good, nm. 

I thought: 

```
x.method({a: {b: {c: "d"}}})
```

copied the whole thing "deeply"? Or did I misunderstand?

Reply:
Well, OK, if the inner things are dictionaries, then _their_ copy semantics get applied, so yeah, that particular example is deep. But `x.method{ a: window })` does not copy `window` deeply.

Reply:
sorry, yeah, I was thinking exclusively about passing dictionaries.